package.json

```diff
@@ -67,7 +67,9 @@
     "test:unit:watch": "npm run test:unit:coverage -- --watchAll",
     "test:e2e": "cross-env DEBUG=test JEST_IMAGE_SNAPSHOT_TRACK_OBSOLETE=1 JEST_PUPPETEER_CONFIG=./test/e2e/jest-puppeteer.config.js jest --runInBand --detectOpenHandles --testPathIgnorePatterns ./test/e2e/performance --config=./test/e2e/jest.config.js",
     "test:perf": "cd ./test/e2e && jest --runInBand --detectOpenHandles --testNamePattern=performance",
-    "test:e2e:coverage": "npm run test:e2e -- --coverage"
+    "test:e2e:coverage": "npm run test:e2e -- --coverage",
+    "version-prepare": "node scripts/manage-version-suffix.mjs && git commit --no-verify -a -m '[RELEASE] prepare version for release'",
+    "postversion": "node scripts/manage-version-suffix.mjs && git commit --no-verify -a -m '[RELEASE] prepare version for new developments'"
```

Comment:
this command seems to not work
fist part(suffix) is OK but git commit no
here I have lunched them in order manually (files are modified but no commit)

```bash
C:\p\bpmn-visu-js>node scripts/manage-version-suffix.mjs
Managing version suffix in configuration files
Current version 0.8.0-post
New version 0.8.0
Configuration files have been updated

C:\p\bpmn-visu-js>git commit --no-verify -a -m '[RELEASE] prepare version for new developments'
fatal: Paths with -a does not make sense.
```

Reply:
do you have the same result when running `npm run version-prepare`?

Reply:
@csouchet could you check what is the behaviour on macOS?
@aibcmars remember that we plan to use this within the GitHub worflow running on Ubuntu, so if we have an issue on Windows, we could live with it :smile_cat: but let's take some time to figure out what is going on

Reply:
### On MacOs
```
➜  BPMN-Js git:(903-Organize_screenshots_of_visual_tests) npm run version-prepare && npm version patch -m '[RELEASE] %s'

> bpmn-visualization@0.8.0-post version-prepare /Users/csouchet/Source/Process Analytics/BPMN-Js
> node scripts/manage-version-suffix.mjs && git commit --no-verify -a -m '[RELEASE] prepare version for release'

Managing version suffix in configuration files
Current version 0.8.0-post
New version 0.8.0
Configuration files have been updated
Auto packing the repository in background for optimum performance.
See "git help gc" for manual housekeeping.
warning: The last gc run reported the following. Please correct the root cause
and remove .git/gc.log.
Automatic cleanup will not be performed until the file is removed.

warning: There are too many unreachable loose objects; run 'git prune' to remove them.

[909_use_dev_version_after_release c217adc] [RELEASE] prepare version for release
 3 files changed, 3 insertions(+), 3 deletions(-)
v0.8.1

> bpmn-visualization@0.8.1 postversion /Users/csouchet/Source/Process Analytics/BPMN-Js
> node scripts/manage-version-suffix.mjs && git commit --no-verify -a -m '[RELEASE] prepare version for new developments'

Managing version suffix in configuration files
Current version 0.8.1
New version 0.8.1-post
Configuration files have been updated
Auto packing the repository in background for optimum performance.
See "git help gc" for manual housekeeping.
warning: The last gc run reported the following. Please correct the root cause
and remove .git/gc.log.
Automatic cleanup will not be performed until the file is removed.

warning: There are too many unreachable loose objects; run 'git prune' to remove them.

[909_use_dev_version_after_release 529a144] [RELEASE] prepare version for new developments
 2 files changed, 2 insertions(+), 2 deletions(-)
```

Reply:
thanks @csouchet 
We have found that the issue cames from the single quote used for the git command message.
Switching to double quote make the script pass on Windows.
Also tested on Ubuntu with `npm run version-prepare && npm run postversion`
Fixed in 70eca5f

Reply:
@csouchet we see that 3 files are updated in 'version-prepare' and only 2 with 'postversion'.
Do you know if this is related to the git warnings that we see prior the commit?

Reply:
The tag always works on MacOs after commit 70eca5f, but the version in **sonar-project.properties** is still not the same as **package.json** after the command running.

Reply:
The problem comes to `npm version patch -m "[RELEASE] %s"` who updates only package.json & package-lock.json with `0.8.1`and create the commit + tag.

I think we should the `version` script, like https://docs.npmjs.com/cli/v6/commands/npm-version, to update the version in the **sonar-project.properties**.

Reply:
Or if it's not possible, disable **git-tag-version** of `npm version`, and create a new script to run after `npm version`, and before the commit and tag of the version change.

Reply:
The removing of the git commit change nothing. The problem is still here 😉